### PR TITLE
[2/n][Runs Feed] Add breadcrumb on backfill run + warning if backfill selected for bulk action

### PR DESCRIPTION
**Pull request by salazarm**
## Summary & Motivation

Screenshot 2024-09-19 at 6 00 54 PM
Screenshot 2024-09-19 at 5 26 45 PM

## Changelog
NOCHANGELOG

### CODE REVIEW

**Comment by salazarm** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24611?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#24621** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24621?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24611** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24611?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#24598** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24598?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @salazarm and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx [hunk 1]
```diff
@@ -153,23 +158,52 @@ export const RunsFeedRoot = () => {
...
             <RunBulkActionsMenu
               clearSelection={() => onToggleAll(false)}
-              selected={entries.filter(
-                (e): e is RunsFeedTableEntryFragment_Run =>
-                  checkedIds.has(e.id) && e.__typename === 'Run',
-              )}
+              selected={selectedRuns}
+              notice={
+                backfillsExcluded ? (
+                  <Alert
```
Comment (hellendag) [inline js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx:192]: Seems a little strange to put a whole `Alert` into a popover menu.

File: js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx [hunk 2]
```diff
@@ -153,23 +158,52 @@ export const RunsFeedRoot = () => {
...
         <Box
           flex={{justifyContent: 'space-between'}}
           style={{width: '100%'}}
           padding={{left: 24, right: 12}}
         >
-          {button}
+          <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}}>
+            {button}
+            <Checkbox
+              label={<span>Hide sub-runs</span>}
```
Comment (hellendag) [inline js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx:178]: What is a sub-run?

**Review by hellendag — APPROVED** [review]

File: js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx (hunk 2 above)
Comment (jamiedemaria) [reply js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx:178]: we may need to come up with a better name for it. It's intended that a "subrun" is any run that is part of a backfill. It originated when we were trying to eliminate the use of "backfill" entirely, but needed a way to still refer to runs that were launched as part of a larger coordinated process

Comment (salazarm) [reply js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx:178]: I think "child runs" would be better?

**Comment by salazarm** [comment]
### Merge activity

* **Sep 23, 2:56 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/24611) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

File: js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx (hunk 2 above)
Comment (jamiedemaria) [reply js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx:178]: yeah maybe. I'd like to get some other opinions on it, I put it as a discussion topic for our sync tomorrow

**Pull request merged**